salt/modules/pkgin.py

```diff
@@ -81,10 +81,17 @@ def _supports_regex():
     '''
     Check support of regexp
     '''
-
     return tuple([int(i) for i in _get_version()]) > (0, 5)
 
 
+@decorators.memoize
+def _supports_parsing():
```

Comment:
As per the change log in 0.7.0 the ```-p``` flag was introduced to
generate parse-able output.

We assume we have this output if we have the regex support but those are
separate features. This commit adds a new internal function to detect the
support. This also fixes latest_version to now work properly on systems
with this newer version of pkgin.